src/patternfly/components/Popover/popover.scss

```diff
@@ -94,11 +129,78 @@
       transform: translateX(var(--pf-c-popover__arrow--m-right--TranslateX)) translateY(var(--pf-c-popover__arrow--m-right--TranslateY)) rotate(var(--pf-c-popover__arrow--m-right--Rotate));
     }
   }
+
+  &.pf-m-icon {
+    display: flex;
```

Comment:
I'm not sure if you need this to change the layout of the popover component itself. This class could be used to target the title via `.pf-c-popover__content > .pf-c-title { display: flex; }` though.

Reply:
Fixed